MonoGame.Framework/SDL/SDL2.cs

```diff
@@ -52,16 +52,20 @@ private static IntPtr GetNativeLibrary()
     public static int Minor;
     public static int Patch;
 
-    private static unsafe string GetString(IntPtr handle)
+    public static unsafe string GetString(IntPtr handle)
     {
         if (handle == IntPtr.Zero)
-            return "";
+            return string.Empty;
```

Comment:
@cra0zy I guess it would be more appropriate to return `null` here. I didn't change it because some code might depend on this behavior. Empty string is right only when `handle` points to `\0`.